utilities/game.py

```diff
@@ -53,18 +52,10 @@ def field_flip_pose2d(p: Pose2d):
     )
 
 
-def field_flip_translation3d(t: Translation3d):
-    return Translation3d(FIELD_LENGTH - t.x, t.y, t.z)
-
-
 def field_flip_rotation2d(r: Rotation2d):
```

Comment:
Note that `field_flip_pose2d` is still used in `utilities/position.py`. We should update that ASAP per #4.